compiler/src/dotty/tools/dotc/config/Feature.scala

```diff
@@ -24,7 +24,7 @@ object Feature:
   private val scala2macros = experimental("macros")
 
   val dependent = experimental("dependent")
-  val erasedTerms = experimental("erasedTerms")
+  val erasedDefinitions = experimental("erasedDefinitions")
```

Comment:
Erased is not only about definitions. It is also about the term values they provide. This is completely lost with this new name. The `erased class` concept is also about erasing term definitions and uses as arguments. Therefore, the scope of the feature has not really changed.

Reply:
The fact that if you erase a definition you also erase its RHS, and if you erase a parameter, you also erase its argument follows naturally since it's the only thing that makes sense. But the trigger is the `erased` on a definition. Or parameter, but I think it's OK to include that in the meaning.